src/helpers/analytics/events/pageFocus.ts

```diff
@@ -1,19 +1,28 @@
-import { createSelector } from 'reselect';
-import * as selectNavigation from '../../../app/navigation/selectors';
+import { stateChange } from '@openstax/event-capture-client/events';
+import { Document } from '@openstax/types/lib.dom';
+import { AppState } from '../../../app/types';
 import { AnalyticsEvent } from './event';
 
-// const eventName = 'REX page focus';
-
-export const selector = createSelector(
-  selectNavigation.pathname,
-  (pathname) => ({pathname})
-);
+export const selector = (_state: AppState) => ({});
```

Comment:
I'm probably overlooking something obvious but do we need to keep this and the unused param in track?

Reply:
yea it needs to match the signature for the other events